src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -411,6 +406,19 @@ public boolean accept(RtpPacket rtpPacket)
         return adaptiveTrackProjection.accept(rtpPacket);
     }
 
+    public boolean transformRtcp(RtcpSrPacket rtcpSrPacket)
+    {
+        long ssrc = rtcpSrPacket.getSenderSsrc();
+        if (ssrc < 0)
+        {
+            return false;
+        }
+
+        AdaptiveTrackProjection adaptiveTrackProjection = adaptiveTrackProjectionMap.get(ssrc);
+
+        return adaptiveTrackProjection != null && adaptiveTrackProjection.rewriteRtcp(rtcpSrPacket);
```

Comment:
Line length

Reply:
line length is consistent with the new rule we voted a while back (120 for all projects)